orc8r/cloud/go/obsidian/swagger/v1/swagger.yml

```diff
@@ -6089,6 +6091,21 @@ paths:
       summary: Update enodebd e2e test case
       tags:
       - e2e
+  /version:
+    get:
+      responses:
+        "200":
+          description: Version
+          schema:
+            example:
+            - 1.6.0
+            - development
+            type: string
+        default:
+          $ref: '#/responses/UnexpectedError'
+      summary: Get version
+      tags:
+      - Version
```

Comment:
I would make this tag "About", so we can expose additional metadata in the future